src/migrator/async.ts

```diff
@@ -28,6 +29,33 @@ import {
 	BaseAsyncMigration,
 } from './utils';
 
+const ASYNC_MIGRATION_ENV_VAR_NAME = 'PINEJS_ASYNC_MIGRATION_ENABLED';
```

Comment:
Can you stick this env var related stuff in https://github.com/balena-io/pinejs/blob/v14.54.1/src/config-loader/env.ts please

Reply:
Yes